### DISCUSSION TITLE
how can i take all league scores for this week and export it to csv

### DISCUSSION THREAD

**Discussion by amirbh1106** [cwendt94/espn-api#300] [Q&A]
how can i take all league scores for this week and export it to csv  thank you for your help

**Comment by cwendt94** [comment]
Not sure exactly the format you would want to use but this seems like a good guide for writing [csv files in python](https://www.pythontutorial.net/python-basics/python-write-csv-file/). You can just get the `league.box_scores()` and then go through each matchup and take the team name and score.
